CHANGELOG.md

```diff
@@ -1,5 +1,102 @@
+* chore(deps): update docker.io/library/golang:1.19.6 docker digest to 7ce31d1 (v1.13) (#23775, @renovate[bot])
+* chore(deps): update docker.io/library/golang:1.19.6 docker digest to 7ce31d1 (v1.13) (#23776, @renovate[bot])
+* chore(deps): update quay.io/cilium/hubble docker tag to v0.11.2 (v1.13) (#23908, @renovate[bot])
+* docs: Clarify basic kernel requirement (Backport PR #24058, Upstream PR #23951, @pchaigno)
+* docs: Document CONFIG_PERF_EVENTS requirement (Backport PR #24200, Upstream PR #24055, @joestringer)
+* docs: Document kernel requirement for L3 devices support (Backport PR #24200, Upstream PR #24101, @pchaigno)
+* docs: Document upgrade behaviour for 1.13.x (#24364, @joestringer)
+* docs: Fix missing disclaimer content to Ingress and Gateway API pages (Backport PR #23956, Upstream PR #23756, @kayceeDev)
+* docs: Fix the dead link to Mellanox performance tuning guide (Backport PR #24086, Upstream PR #24012, @gentoo-root)
+* docs: replace usage of api.twitter.com (Backport PR #23834, Upstream PR #23669, @kaworu)
+* Document exemplars option for hubble httpV2 metrics (Backport PR #23834, Upstream PR #23620, @chancez)
+* fix(deps): update module golang.org/x/net to v0.7.0 [security] (master) (Backport PR #23956, Upstream PR #23904, @renovate[bot])
+* Fixed broken/deprecated links (Backport PR #24058, Upstream PR #23920, @PhilipSchmid)
+* Fixed link to broken anchor in RKE doc (Backport PR #23834, Upstream PR #23706, @raphink)
+* Fixes a flake in the kubectl wait part of the CI (Backport PR #23834, Upstream PR #23733, @meyskens)
+* IPsec: Remove `IP_POOLS` logic (Backport PR #24086, Upstream PR #24030, @pchaigno)
+* kvstore: add clusterName suffix to session controllers (Backport PR #24086, Upstream PR #23928, @oblazek)
+* Remove / in RKE doc link as it causes redirect bug (Backport PR #23834, Upstream PR #23728, @raphink)
+* test/runtime: Set NO_COLOR for privileged tests (Backport PR #24058, Upstream PR #23151, @joestringer)
+* Update CNI to 1.2.0 (#23319, @michi-covalent)
+* Update signature verification docs for Sigstore 2.0 (Backport PR #24086, Upstream PR #24029, @jedsalazar)
+* workflow: fixes LLVM, Clang cache and install path (Backport PR #23834, Upstream PR #23740, @brlbil)
+
+**Other Changes:**
+* .github: remove workflows that are not branch specific (#23842, @aanm)
+* [v1.13] bpf: use skb->ifindex for FIB lookup in handle_*_from_lxc() (#24195, @julianwiedmann)
+* gha: Bump timeout to 90 minutes for build commit. (#23959, @sayboras)
+* install: Update image digests for v1.13.0 (#23783, @aanm)
+* update images 1.13 (#24331, @nebril)
+* v1.13 - Backport initContainer change (#24333, @ferozsalam)
+* v1.13 backport: fix cgroup program detachment and 1.14 downgrade (#24184, @ti-mo)
+* v1.13 Backports 2023-03-06 (#24179, @jibi)
+* v1.13 Backports 2023-03-14 (#24370, @nebril)
+* v1.13 Backports 2023-03-15 (#24387, @nebril)
```

Comment:
Why didn't these pull the underlying PRs? We may be missing release notes due to this.

Reply:
It might be because there are two ```` ```...``` ```` quote sections in the PR messages for these PRs, I'm not sure if the release tool understands how to handle that format.